.travis.yml

```diff
@@ -1,18 +1,55 @@
+sudo: required
```

Comment:
It looks like the run time went from ~3-4 minutes to ~5-7 minutes. I really want Travis to be < 5 minutes for a change, and we've already been creeping up slowly, this one jumps up significantly.

Since 99% of TChannel PRs don't touch tracing, I don't want every Travis run for each small change to get significantly slower without a large benefit.